### PR TITLE
Added background to AudioButton,VideoButton(converted to AppCompatImageButton)

### LINKED ISSUE #577: Audio button has no background
#### Software and hardware versions 
Current master branch

#### Problem description
Audio button has no background.

#### Steps to reproduce the problem
1. Open "sample" form.
2. Go to the first question

#### Expected behavior
Audio button should look like before.

#### Other information 
It's coused by changes from #525:
[compileSdkVersion](https://github.com/opendatakit/collect/pull/525/files#diff-fa0f36229667501896e9f29c3a2a4b35R60)
[AudioButton](https://github.com/opendatakit/collect/pull/525/files#diff-ce9802d4c0f435bd3f26034d56b61e95R39)
**BEFORE:**
before
**NOW:**
now

@lognaturel you can mark this issue as "quick win".

### PR DESCRIPTION
**Pull request by pratikmjoshi**
Fixes #577 
Kept AudioButton as AppCompatImageButton, made VideoButton AppCompatImageButton, and added background to both,

### CODE REVIEW

**Comment by pratikmjoshi** [comment]
@lognaturel @grzesiek2010 I have made the necessary changes. Using ./gradlew build, the app is building successfully. But with circleci, I seem to be getting a timeout, which maybe linked to issue #593. Could you please look into it?

**Comment by grzesiek2010** [comment]
@pratikmjoshi We have a problem with CircleCI so no worries as long as everything is ok when you use "./gradlew build" command keep it up. At firs glance it looks ok but prs should be focused as much as possible so please remove all other changes.If you want to refactor the code you can do that of courese in separate pr. It would be great if you can attach screenshots as well.

**Comment by pratikmjoshi** [comment]
@grzesiek2010 Oh,I didn't mean to put the refactoring changes in this pr. Should I close this pr and submit a new one? I'll attach screenshots in that pr.

**Comment by grzesiek2010** [comment]
@pratikmjoshi it's not necessary to close this pr you can remove unnecessary changes or if it's easier for you - you can start with a new branch and then you can use:

> git push origin newaudiobutton --force

and everything should be ok.

**Comment by pratikmjoshi** [comment]
@grzesiek2010 I think some of my commit messages have been associated with wrong commits.  I think its best if I close this pr and start a new one. I've accidentally put a junk commit message in there too("ok"). I'll put the screenshots in the next one. Sorry for the inconvenience(that too for a simple problem).

**Comment by grzesiek2010** [comment]
@pratikmjoshi as I said you don't need to close this pr :) Even if you start with a new branch for example your current branch as I see is called newaudiobutton. You can create a new branch (starting from master branch) eg newaudiobutton2 then you can add all required changes and use the comand I mentioned above in that case you would need:

> git push origin newaudiobutton2:newaudiobutton --force

and this pr will be replaced.

You can of course as well - go to master branch:

> git checkout master

then remove your branch:

> git branch -D newaudiobutton

and creat a new branch with the same name:

> git checkout -b newaudiobutton

then you need only:

> git push origin newaudiobutton --force

to push your changes and update pr

**Comment by pratikmjoshi** [comment]
@grzesiek2010 Oh perfect. Thanks! All other changes have been removed. Just the proposed changes have been made. Screenshot below:

screenshot_20170315-015128

File: collect_app/src/main/java/org/odk/collect/android/views/MediaLayout.java [hunk 1]
```diff
@@ -61,7 +62,7 @@
     private FormIndex mIndex;
     private TextView mView_Text;
     private AudioButton mAudioButton;
-    private ImageButton mVideoButton;
```
Comment (grzesiek2010) [inline collect_app/src/main/java/org/odk/collect/android/views/MediaLayout.java:64]: I bet there is unused import android.widget.ImageButton as you implemented this change :) Please remove that if I'm right.

**Comment by grzesiek2010** [comment]
@pratikmjoshi @lognaturel Looks ok :+1:

**Comment by lognaturel** [comment]
Thank you @pratikmjoshi @grzesiek2010

**Pull request merged**